docs/Supported-Platforms.md

```diff
@@ -0,0 +1,82 @@
+# Supported Platforms
+
+- [Summary](#summary)
+- [Per-Platform Details](#per-platform-details)
+  * [Windows](#windows)
+  * [macOS](#macos)
+  * [Linux](#linux)
+
+# Summary
+
+[Downloadable packages](https://www.brimsecurity.com/download/) for Brim are
+available that run on Windows, macOS, and Linux. The specific
+versions/distributions of these platforms for which we can set expectations of
+quality are affected by end-of-life cycles and their availability in our
+automated test infrastructure.
+
+The sections below describe the details regarding the versions we test with
+continuously as well as occasional ad hoc testing. We welcome you to
+[open an issue](Troubleshooting.md#opening-an-issue) on any problem you
+experience with Brim regardless of platform version, but please understand
+that we may be limited in our ability to provide quick fixes (or any fix at
+all) for certain platforms, subject to availability.
+
+# Per-Platform Details
+
+The following details are current as of Brim `v0.9.0`.
```

Comment:
Could we change this to say that it reflects the current latest Brim version, w/o specifying the version number? I don't want to us need to update this for every release.

Reply:
Dropped the sentence.